python/pyarrow/tests/test_ipc.py

```diff
@@ -590,7 +590,8 @@ def test_read_options_included_fields(stream_fixture):
     assert result1 == table
 
 
-def test_dictionary_delta(format_fixture):
+def test_dictionary_delta(format_fixture, request):
+    format_fixture = request.getfixturevalue(format_fixture)
```

Comment:
I would move it here to the format_fixture as well, just to keep it as close to the definition